actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -280,16 +285,19 @@ private void createOrUpdateOrganisations(
     }
   }
 
-  private void removeOrganisations(Map<String, Object> orgDbMap, String rootOrgId) {
+  private void removeOrganisations(
+      Map<String, Object> orgDbMap, String rootOrgId, String requestedBy) {
     Set<String> ids = orgDbMap.keySet();
     UserOrgDao userOrgDao = UserOrgDaoImpl.getInstance();
     ids.remove(rootOrgId);
-    List<String> userOrgTobeRemoved = new ArrayList<>();
     for (String id : ids) {
-      userOrgTobeRemoved.add((String) ((Map<String, Object>) orgDbMap.get(id)).get(JsonKey.ID));
-    }
-    if (CollectionUtils.isNotEmpty(userOrgTobeRemoved)) {
-      userOrgDao.deleteUserOrgs(userOrgTobeRemoved);
+      UserOrg userOrg = mapper.convertValue(orgDbMap.get(id), UserOrg.class);
+      userOrg.setDeleted(true);
+      userOrg.setId((String) ((Map<String, Object>) orgDbMap.get(id)).get(JsonKey.ID));
+      userOrg.setUpdatedDate(ProjectUtil.getFormattedDate());
```

Comment:
remove deleteUserorgs function from userOrgDao  - as we are not supporting only soft delete